tests/services/auth.service.test.ts

```diff
@@ -34,4 +35,71 @@ describe('authService', () => {
       expect(logSpy).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('login', () => {
+    fit('Should return a JWT token if the given credentials are correct', async () => {
+      const credentials: AuthCredentials = {
+        email: 'willem.wortel@icapps.com',
+        password: 'developer',
+      };
+
+      const tokenObject =  await login(credentials, 'jwt');
+      console.log(typeof(tokenObject));
+      expect(tokenObject).toHaveProperty('accessToken');
+
+    });
+
+    it('Should throw an error when the username is correct but the password is not', async () => {
+      const credentials: AuthCredentials = {
+        email: 'willem.wortel@icapps.com',
+        password: 'wrongpassword',
+      };
+      expect.assertions(2);
+      try {
+        await login(credentials, 'jwt');
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error);
+        expect(error.message).toEqual('Incorrect username or password. Please try again.');
+      }
+    });
+
+    it('Should throw an error when the user does not exist', async () => {
+      const credentials: AuthCredentials = {
+        email: 'notexisting@icapps.com',
+        password: 'developer',
+      };
+      expect.assertions(2);
+      try {
+        await login(credentials, 'jwt');
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error);
+        expect(error.message).toEqual('Incorrect username or password. Please try again.');
+      }
+    });
+  });
+
+  describe('logout', () => {
+    it('Should throw an error when it can not destroy the session', async () => {
+      // TODO: check if this is a good test...
+      const req = {
+        // no session
+      };
+      try {
+        await logout(req);
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error);
+      }
+    });
+
+    it('Should succesfully destroy the session', () => {
+      // TODO: implement
```

Comment:
Implement?

Reply:
export async function logout(req) {
  return new Promise((resolve, reject) => {
    req.session.destroy((error) => {
      if (error) {
        logger.error(`An error occured trying to logout: %${error}`);
        return reject(error);
      }
      return resolve();
    });
  });
}

Is this neccesary to test? Since the logic is pure express-session?